lem.asd

```diff
@@ -257,7 +257,8 @@
 
 (defsystem "lem/legit"
   :serial t
-  :depends-on ("lem" "lem-patch-mode" "lem-yaml-mode" "lem-markdown-mode")
+  :depends-on ("lem" "lem-patch-mode" "lem-yaml-mode" "lem-markdown-mode"
+               "cl-git" "split-sequence" "arrow-macros")
```

Comment:
Now legit depends on `libgit2-dev` on Debian, and on other libs for other systems. We need at least a check and a user-friendly message if it doesn't exist.

Now lem depends on libgit2, we'll need to enhance the installation scripts, the Docker image, the qlfile…

Shall the use of cl-git be optional, activated only if libgit2 is present?

Reply:
At the very least you need to add a note in the README ;)

    apt install libgit2-dev